src/pages/Integrations/Create/helpers.tsx

```diff
@@ -54,9 +54,13 @@ export const nextDetailsStep = (values) => {
     return 'slack-details';
   } else if (
     values['integration-type'] === IntegrationType.TEAMS ||
-    IntegrationType.GOOGLE_CHAT
+    values['integration-type'] === IntegrationType.GOOGLE_CHAT
   ) {
     return 'teams-gchat-details';
+  } else if (values['integration-type'] === IntegrationType.SPLUNK) {
```

Comment:
Ideally we'll utilize DDF conditional steps https://www.data-driven-forms.org/examples/custom-wizard#heading-customwizard